lib/private/Setup.php

```diff
@@ -542,19 +542,27 @@ public static function protectDataDirectory() {
 		//Require all denied
 		$now =  date('Y-m-d H:i:s');
 		$content = "# Generated by Nextcloud on $now\n";
-		$content.= "# line below if for Apache 2.4\n";
+		$content.= "# Section for Apache 2.4 and 2.5\n";
```

Comment:
```suggestion
		$content.= "# Section for Apache 2.4 and 2.6\n";
```

http://httpd.apache.org/dev/devnotes.html

Reply:
@kesselb 
So do I get it right that 2.5 will be finally released as 2.6?
Should be changed in the autoindex part below as well then.

Reply:
I think so